src/pieces/api/pieces_websocket.py

```diff
@@ -6,6 +6,7 @@
 from rich.live import Live
 from rich.markdown import Markdown
 from . import config
+import pieces_os_client as pos_client
 
 WEBSOCKET_URL = "ws://localhost:1000/qgpt/stream"
```

Comment:
Looking good, I believe I originally wrote the websocket url but we should probably change it at some point to use the sdk instead of localhost

Reply:
Good point we need to change it because on Linux the port differs

Reply:
yeah will want to use the linux port here if on linux good catch

Reply:
https://github.com/pieces-app/cli-agent/pull/79/commits/83c04e0cc7a9604ad4cc4b6cd18fdea2ec3552ae